src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -120,6 +120,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@SteveL-MSFT, your last commit had 25 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Cdxml cmdlets are supported.Module level tests.The CimTest module should have been loaded
```powershell
MofComp.exe failed with exit code 1
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Cdxml\Cdxml.Tests.ps1: line 98
98:             throw "MofComp.exe failed with exit code $MofCompReturnCode"

```
Cdxml cmdlets are supported.Module level tests.The CimTest module should have the proper cmdlets
```powershell
MofComp.exe failed with exit code 1
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Cdxml\Cdxml.Tests.ps1: line 98
98:             throw "MofComp.exe failed with exit code $MofCompReturnCode"

```
Cdxml cmdlets are supported.Get-CimTest cmdlet.The Get-CimTest cmdlet should return 4 objects
```powershell
MofComp.exe failed with exit code 1
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Cdxml\Cdxml.Tests.ps1: line 98
98:             throw "MofComp.exe failed with exit code $MofCompReturnCode"

```
Cdxml cmdlets are supported.Get-CimTest cmdlet.The Get-CimTest cmdlet should retrieve an object via id
```powershell
MofComp.exe failed with exit code 1
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Cdxml\Cdxml.Tests.ps1: line 98
98:             throw "MofComp.exe failed with exit code $MofCompReturnCode"

```
Cdxml cmdlets are supported.Get-CimTest cmdlet.The Get-CimTest cmdlet should retrieve an object by piped id
```powershell
MofComp.exe failed with exit code 1
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Cdxml\Cdxml.Tests.ps1: line 98
98:             throw "MofComp.exe failed with exit code $MofCompReturnCode"

```